### PR TITLE
Team Beta Submission

### PR DESCRIPTION
**Pull request by rohansheelvant**
Team Beta Submission Assignment 1

### CODE REVIEW

File: services/backend/src.zip
Comment (kennethgoodman) [inline services/backend/src.zip:1]: please remove

File: Report 1.pdf
Comment (kennethgoodman) [inline Report 1.pdf:1]: pdf to be uploaded to canvas please

File: services/backend/src/config.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ class BaseConfig:
     BCRYPT_LOG_ROUNDS = 13
     ACCESS_TOKEN_EXPIRATION = 900  # 15 minutes
     REFRESH_TOKEN_EXPIRATION = 2592000  # 30 days
-    NUMBER_OF_CONTENT_IN_ANN = 100 # UPDATE THIS WHEN DEVELOPING ANN
```
Comment (kennethgoodman) [inline services/backend/src/config.py:11]: please remove, this is for your own development only

File: services/backend/src/config.py [hunk 2]
```diff
@@ -28,5 +28,5 @@ class TestingConfig(BaseConfig):
 class ProductionConfig(BaseConfig):
     SQLALCHEMY_DATABASE_URI = os.environ.get("DATABASE_URL", "")
     SECRET_KEY = os.getenv("SECRET_KEY", "")
-    NUMBER_OF_CONTENT_IN_ANN = 1000
```
Comment (kennethgoodman) [inline services/backend/src/config.py:31]: same here

File: services/backend/src/data_structures/user_based_recommender/beta/UserBasedRecommender.py [hunk 3]
```diff
@@ -3,39 +3,91 @@
...
         if cls._instance is None:
+            #print("init function")
             cls._instance = super(UserBasedRecommender, cls).__new__(cls)
             cls._instance.user_similarity_map = {}
             cls._instance.gather_data()
             cls._instance.compute_similarity()
         return cls._instance
 
     def gather_data(self):
+        #print("gather data function")
```
Comment (kennethgoodman) [inline services/backend/src/data_structures/user_based_recommender/beta/UserBasedRecommender.py:26]: remove commented code please

File: services/db/01_user.sql [hunk 4]
```diff
@@ -1,18 +1,3 @@
...
-/*!40000 ALTER TABLE `user` ENABLE KEYS */;
-UNLOCK TABLES;
\ No newline at end of file
+version https://git-lfs.github.com/spec/v1
```
Comment (kennethgoodman) [inline services/db/01_user.sql:1]: please undo this change

File: services/backend/src.zip
Comment (rohansheelvant) [reply services/backend/src.zip:1]: Ok professor

File: Report 1.pdf
Comment (rohansheelvant) [reply Report 1.pdf:1]: Ok, done

File: services/backend/src/config.py (hunk 1 above)
Comment (jonathan80622) [reply services/backend/src/config.py:11]: Thank you professor for pointing out. Will this number be increased when it is served in production?

File: services/db/01_user.sql (hunk 4 above)
Comment (rohansheelvant) [reply services/db/01_user.sql:1]: Hi Professor, I am unable to modify this.

I downloaded the original file from your repo and added that to my local. However, I am not able to commit it as it says my branch is up to date with all the files. I tried to open this file locally and check what is inside it, it is the same as in your file. Can you please help me here?

**Comment by rohansheelvant** [comment]
Hi Professor,

Do we need to make any other changes as well?

**Comment by kennethgoodman** [comment]
please remove changes to config.py

**Comment by rohansheelvant** [comment]
Done professor

**Review by kennethgoodman — APPROVED** [review]
